R/sensitivityCalculation.R

```diff
@@ -61,6 +61,10 @@ sensitivityCalculation <- function(simulation,
   variationRange <- .validateVariationRange(variationRange)
 
   # creating `SimulationResults` batch ------------------------
+  # Store old simulation outputs and set user defined
+  oldOutputSelections <- simulation$outputSelections$allOutputs
```

Comment:
This was present in the original implementation but has been removed for some reason.

Reply:
Like you had told me to do, I completely discarded the original implementation and started afresh on my own.

The PR should have been merged after #175 was resolved. It'd have caught that this part was missing.